utt/include/globalParams.hpp

```diff
@@ -20,27 +20,32 @@ class Params;
 namespace libutt::api {
 class GlobalParams {
   /**
-   * @brief Represents a shared known global UTT params. These parameters includes commitment keys, nullifier parametrs
+   * @brief Represents a shared known global UTT params. These parametesr include commitment keys, nullifier parameters
    * and more (see uttlib/Params.h)
    *
    */
-  static bool initialized;
 
  public:
+  struct BaseLibsInitData {
+    std::string ntl_finite_field{"21888242871839275222246405745257275088548364400416034343698204186575808495617"};
```

Comment:
ntl_finite_field should not be a variable. This number is the same big prime used with the bn128 curve that libff uses. This number should not be a part of an interface unless we allow customization of the underlying cryptographic math, which we probably wont do for quite some time.